Add 008 network tests (#160)
tqtezos/stablecoin: haskell/README.md

The help message should be sufficiently descriptive, however, there are some caveats:
1. [`tezos-client`](http://tezos.gitlab.io/introduction/howtoget.html) executable must be available and is assumed to be in `$PATH` by default.

>    Note that only `tezos-client` from version v8.0 is supported.

Reviewer: Nitpick rephrasing.
```suggestion
   Note that only `tezos-client` versions v8.0 or above are supported.
```
